compiler/IREmitter/Exceptions.cc

```diff
@@ -119,17 +149,24 @@ class ExceptionState {
         auto [bodyResult, exceptionResultPtr] = sorbetTry(bodyFunction, previousException);
 
         // Check for an early return from the body.
-        auto *earlyReturnBlock = llvm::BasicBlock::Create(cs, "exception-body-return", currentFunc);
-        auto *continueBlock = llvm::BasicBlock::Create(cs, "exception-body-continue", currentFunc);
+        auto *earlyReturnBlock = this->bodyReturn;
+        auto *continueBlock = this->handlersBlock;
```

Comment:
Is it worth deleting these declarations and inlining the fields from the rhs?

Reply:
I wondered about this when I was writing the patch, but ultimately decided to keep them for the purposes of this patch to try and minimize the diff.  I am totally willing to delete "useless" declarations as a followup.